java-extras/src/test/java/org/triplea/java/concurrency/AsyncRunnerTest.java

```diff
@@ -0,0 +1,62 @@
+package org.triplea.java.concurrency;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsInstanceOf.instanceOf;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.CompletionException;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class AsyncRunnerTest {
+
+  @Mock private Runnable runnable;
+  @Mock private Consumer<Throwable> exceptionHandler;
+  private final ArgumentCaptor<Throwable> exceptionArgumentCaptor =
+      ArgumentCaptor.forClass(Throwable.class);
+
+  @Test
+  void runASync() {
```

Comment:
The case of "Async" is inconsistent with all the other test names

Reply:
Eagle eye :eagle:  :+1: